dockstore-integration-testing/src/test/java/io/dockstore/client/cli/SystemClientIT.java

```diff
@@ -97,9 +100,11 @@ private static ApiClient getWebClient(boolean correctUser, boolean admin) throws
         File configFile = FileUtils.getFile("src", "test", "resources", "config");
         INIConfiguration parseConfig = Utilities.parseConfig(configFile.getAbsolutePath());
         ApiClient client = new ApiClient();
+        ApiKeyAuth bearer = (ApiKeyAuth)client.getAuthentication("BEARER");
+        bearer.setApiKeyPrefix("BEARER");
+        bearer.setApiKey((correctUser ? parseConfig
+            .getString(admin ? Constants.WEBSERVICE_TOKEN_USER_1 : Constants.WEBSERVICE_TOKEN_USER_2) : "foobar"));
```

Comment:
These 4 lines are mostly likely not needed.

Reply:
Nope, in this suite 10 tests pass, 12 fail without this.

Reply:
I just check this branch out and it passed.

Reply:
I just tried this out as well for the above comment. 
Are you sure you're running integration tests?